### ISSUE TITLE
Automatic deploy suddenly not working

### ISSUE THREAD

**Issue by johnknoop** [projectkudu/kudu#2242]
Hi!

I recently switched from Standard to Free plan of my app service, and since then the automatic deployment has stopped working.

Azure is watching BitBucket for changes, and then runs the deploy. All the steps (restore NuGet packages, build, restore NPM packages, run gulp) are completing, but then, when the actual deploy should take place, it just stops with these messages:

Omitting next output lines...
Processed 563 files...
Processed 1158 files...
Processed 1765 files...
Processed 2481 files...
Thread was being aborted.
Thread was being aborted.

Any ideas?

**Comment by davidebbo** [comment]
I all likelihood, your free site is running out of CPU quota during the build. Can you check the quotas in the Azure Portal?

**Comment by johnknoop** [comment]
![image](https://cloud.githubusercontent.com/assets/3394270/20530717/87e10164-b0d4-11e6-9190-f7ac0c1a1f83.png)

This is how it looks right now (not running any deployments at the moment).

**Comment by johnknoop** [comment]
Or should I check it while running? Maybe the 3-minute quota will spike during deployment?

**Comment by davidebbo** [comment]
Yes, that's likely the quota that's getting exceeded, and you'll need to check right as it's happening.

**Comment by johnknoop** [comment]
![image](https://cloud.githubusercontent.com/assets/3394270/20531581/7c14e5f0-b0d7-11e6-9006-7ad303510431.png)

Seems you're right.

Is there no way to enable "elastic" scaling, so that it may eat more CPU resources, and just put it on the bill?

**Comment by johnknoop** [comment]
Thanks
